app/controllers/items_controller.rb

```diff
@@ -332,11 +332,6 @@ def tags_client
     Dor::Services::Client.object(@cocina.externalIdentifier).administrative_tags
   end
 
-  # Filters
```

Comment:
`save_and_reindex` still refers to `@object`, but `save_and_reindex` looks like it can be excised also, since nothing seems to call it anymore

Reply:
Yes, good call. I removed it.

Reply:
thanks!